wiki/Tournaments/NDC/2022/en.md

```diff
@@ -0,0 +1,504 @@
+| 30 | ::{ flag=NL }:: [OldEclipse](https://osu.ppy.sh/users/5174602) | #4330 | #54 |
+| 31 | ::{ flag=NL }:: [TheDurpyOwl](https://osu.ppy.sh/users/11187258) | #7518 | #95 |
+| 32 | ::{ flag=NL }:: [UnveiledGolem](https://osu.ppy.sh/users/14090438) | #30867 | #404 |
+| 33 | ::{ flag=NL }:: [Kut](https://osu.ppy.sh/users/3859899) | #14821 | #186 |
+| 34 | ::{ flag=NL }:: [Finny666](https://osu.ppy.sh/users/17271135) | #77607 | #1000 |
+| 35 | ::{ flag=NL }:: [Woebel](https://osu.ppy.sh/users/8454542) | #61120 | #792 |
+| 36 | ::{ flag=NL }:: [AintGotNoAim](https://osu.ppy.sh/users/18998391) | #137452 | #1656 |
+| 37 | ::{ flag=NL }:: [przemo52](https://osu.ppy.sh/users/11170330) | #64337 | #835 |
+| 38 | ::{ flag=NL }:: [Dutchi](https://osu.ppy.sh/users/15629640) | #70267 | #910 |
+| 39 | ::{ flag=NL }:: [TheMad_N](https://osu.ppy.sh/users/10523988) | #81868 | #1046 |
+| 40 | ::{ flag=NL }:: [Cookie_Tree](https://osu.ppy.sh/users/502722) | #124124 | #1520 |
+| 41 | ::{ flag=NL }:: [Maarten](https://osu.ppy.sh/users/24414822) | #205802 | #2425 |
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=NL }:: [Viveliam](https://osu.ppy.sh/users/3506793) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=NL }:: [Lilily](https://osu.ppy.sh/users/6502403) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=NL }:: [Skyrovania](https://osu.ppy.sh/users/4696315) |
+
+## Mappools
+
+### Grand Finals
+
+**[Download the mappack here!](https://tourney.huismetbenen.nl/10/mappools/gf)**
+
+- Nomod
+  1. [Fuki Commune - Kurui Zake Setsugekka (-Kirigiri) [Destiny]](https://osu.ppy.sh/beatmapsets/1457529#osu/2995484)
+  2. [pm04034 - Sugary Mocaccino (Astronic) [Caffinated Adrenaline]](https://osu.ppy.sh/beatmapsets/1592198#osu/3251854)
+  3. [Yousei Teikoku - Autoscopy (moph) [Extreme]](https://osu.ppy.sh/beatmapsets/1633220#osu/3333699)
+  4. [MYUKKE. - NO ONE YES MAN (captin1) [PEACE]](https://osu.ppy.sh/beatmapsets/1687287#osu/3448347)
+  5. [Cattle Decapitation - Your Disposal (Chanci) [Eradication]](https://osu.ppy.sh/beatmapsets/988109#osu/2207855)
+  6. [Nekomata Master - Byakuya Gentou (Feiri) [Expert]](https://osu.ppy.sh/beatmapsets/1704624#osu/3483161)
+  7. [Jun.A - The Refrain of the Lovely Great War (Duftende Pizza) [SOLO]](https://osu.ppy.sh/beatmapsets/725650#osu/1532151)
+- Hidden
+  1. [ITO KASHITARO - Yuudachi-no Ribbon (Yudragen) [Flash of Eternity]](https://osu.ppy.sh/beatmapsets/1383608#osu/2858218)
+  2. [Yasui Yousuke - YO-KAI Disco [MEIKAI ARRANGE VERSION] ([Keqing]) [aww shidd]](https://osu.ppy.sh/beatmapsets/1485831#osu/3046747)
+  3. [Chroma - Sudden Visitor (Fisky) [MAXIMUM]](https://osu.ppy.sh/beatmapsets/1251674#osu/2601556)
+  4. [REDALiCE - DEAD or DIE (Cyberia950) [Extra]](https://osu.ppy.sh/beatmapsets/1570534#osu/3207055)
+- Hardrock
```

Comment:
Needs to be `Hard Rock` instead of `Hardrock` throughout all 3 wiki pages

Reply:
This is the first time in over 10 years I realize it's Hard Rock and not Hardrock